api/v1alpha1/utils/vaultobject.go

```diff
@@ -80,3 +80,36 @@ func (ve *VaultEndpoint) CreateOrUpdate(context context.Context) error {
 	}
 	return nil
 }
+
+type VaultObjectWithLease interface {
```

Comment:
This endpoint configures the lease settings for generated credentials from https://www.vaultproject.io/api-docs/secret/rabbitmq#configure-lease.
I tried to use tune, but it seems to have no effect. I set tune default_lease_ttl to 80000s and lease TTL is 86400, but when I generate credentials there "lease_duration" is 86400s.
I'm creating a separate object to be able to set separate path and values, so it will be updated as part of a single reconcile and TTL values can be provided as part of engine configuration.